kitsune/products/jinja2/products/includes/topic_macros.html

```diff
@@ -63,30 +63,6 @@ <h3 class="card--title">
     </div>
   </div>
 </section>
-
-{% if settings.OIDC_ENABLE and product.id in subscribed_products_ids %}
```

Comment:
We should be able to remove the `subscribed_products_ids` key from the context as well:
https://github.com/mozilla/kitsune/blob/9ea6dbb02be71dad4bce92f8d4c13226fcd3a26d/kitsune/products/views.py#L63